### PR TITLE
Moved everything from SandboxCommonEvents and SandboxClientEvents to the individual classes

### PR DESCRIPTION
**Pull request by MsRandom**

### CODE REVIEW

File: base/src/main/java/org/sandboxpowered/api/entity/EntitySize.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package org.sandboxpowered.api.entity;
+
+public interface EntitySize {
```
Comment (Boundarybreaker) [inline base/src/main/java/org/sandboxpowered/api/entity/EntitySize.java:3]: TODO on this?

Comment (MsRandom) [reply base/src/main/java/org/sandboxpowered/api/entity/EntitySize.java:4]: Added

**Comment by TheCodedOne** [comment]
Maybe EntityEvents/BlockEvents etc would be better than in the content class, later down the line this could fill classes with a lot of fields that arent useful to them.

**Comment by MsRandom** [comment]
> Maybe EntityEvents/BlockEvents etc would be better than in the content class, later down the line this could fill classes with a lot of fields that arent useful to them.

Sounds good, should they be BlockEvents, EntityEvents, LivingEvents, and PlayerEvents?

**Comment by TheCodedOne** [comment]
Living should just go into Entity

**Comment by MsRandom** [comment]
That would be 14 fields in EntityEvents though(from ItemEntity, LivingEntity, and Entity)

**Comment by TheCodedOne** [comment]
That's fine

**Comment by MsRandom** [comment]
Done, I kept the content registering event in the Registry class though, but I can change that if needed

File: base/src/main/java/org/sandboxpowered/api/entity/Entity.java [hunk 2]
```diff
@@ -2,9 +2,16 @@
...
+import org.sandboxpowered.api.events.args.SizeUpdateArgs;
 import org.sandboxpowered.api.registry.Registry;
 import org.sandboxpowered.api.util.Mono;
 import org.sandboxpowered.api.util.annotation.Alpha;
+import org.sandboxpowered.eventhandler.EventHandler;
+import org.sandboxpowered.eventhandler.PriorityEventHandler;
+import org.sandboxpowered.eventhandler.core.EventArgs;
+import org.sandboxpowered.eventhandler.core.EventHandlerBase;
+import org.sandboxpowered.eventhandler.priority.Cancellable;
+import org.sandboxpowered.eventhandler.priority.PriorityHandler;
```
Comment (TheCodedOne) [inline base/src/main/java/org/sandboxpowered/api/entity/Entity.java:14]: These aren't needed

File: base/src/main/java/org/sandboxpowered/api/block/Block.java [hunk 3]
```diff
@@ -18,6 +19,8 @@
 import org.sandboxpowered.api.util.math.Vec3f;
 import org.sandboxpowered.api.world.World;
 import org.sandboxpowered.api.world.WorldReader;
+import org.sandboxpowered.eventhandler.PriorityEventHandler;
+import org.sandboxpowered.eventhandler.priority.PriorityHandler;
```
Comment (TheCodedOne) [inline base/src/main/java/org/sandboxpowered/api/block/Block.java:23]: These aren't needed

File: base/src/main/java/org/sandboxpowered/api/block/Block.java [hunk 4]
```diff
@@ -7,6 +7,7 @@
 import org.sandboxpowered.api.entity.Entity;
 import org.sandboxpowered.api.entity.player.Hand;
 import org.sandboxpowered.api.entity.player.PlayerEntity;
+import org.sandboxpowered.api.events.args.BlockArgs;
```
Comment (TheCodedOne) [inline base/src/main/java/org/sandboxpowered/api/block/Block.java:10]: This isn't needed

File: base/src/main/java/org/sandboxpowered/api/entity/Entity.java (hunk 2 above)
Comment (MsRandom) [reply base/src/main/java/org/sandboxpowered/api/entity/Entity.java]: Fixed, I forgot to organize imports earlier when I moved the fields

**Pull request merged**